example.env

```diff
@@ -2,3 +2,4 @@
 ADMIN_USER="define your username"
 ADMIN_PASSWORD="define your password"
 MONGO_HOST="mongo"
+RATE_LIMIT=integer
```

Comment:
Can we change this to may be - RATE_LIMIT="define the number of requests per 10 seconds"